src/views/registration-js/attendees/fields/birth.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * This template renders a Single Ticket content
+ * composed by Title and Description currently
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/registration-js/attendees/fields/birth.php
+ *
+ * @since   TBD
+ * @version TBD
+ *
+ * @see     Tribe__Tickets_Plus__Meta__Field__Birth
+ *
+ */
+$required    = isset( $field->required ) && 'on' === $field->required ? true : false;
+$option_id   = "tribe-tickets-meta_{$field->slug}_{$ticket->ID}{{data.attendee_id}}";
+$birth_field = $field;
+$field       = (array) $field;
+$field_name  = 'tribe-tickets-meta[' . $ticket->ID . '][{{data.attendee_id}}][' . esc_attr( $field['slug'] ) . ']';
+$disabled    = false;
+$classes     = [ 'tribe-common-b1', 'tribe-field', 'tribe-tickets__item__attendee__field__birth' ];
+
+if ( $required ) {
+	$classes[] = 'tribe-tickets-meta-required';
+}
+?>
+<div class="tribe-horizontal-date-picker-container">
+	<div <?php tribe_classes( $classes ); ?> >
+		<label
+				class="tribe-common-b2--min-medium tribe-tickets-meta-label"
+				for="<?php echo esc_attr( $birth_field->month_id ); ?>"
+		><?php echo wp_kses_post( $field['label'] ); ?><?php tribe_required_label( $required ); ?></label>
+
+		<div class="tribe_day_month_year_datepicker tribe__tickets__item__attendee__field__birth__month">
```

Comment:
do we have a convention for class naming here?

Reply:
Done. Thanks! https://github.com/moderntribe/event-tickets/pull/1656/commits/2edcfa1f979b1926a64b8e1915d0c04884bdabf2